src/components/shared/MembersList/MembersList.tsx

```diff
@@ -1,48 +1,45 @@
 import React, { ReactNode } from 'react';
 
 import ListGroup, { ListGroupAppearance } from '~shared/ListGroup';
-import { AnyUser, Colony } from '~data/index';
+import { Watcher, Contributor } from '~types';
 
 import MembersListItem from './MembersListItem';
+import { notNull } from '~utils/arrays';
 
-interface Props<U> {
-  colony: Colony;
-  extraItemContent?: (user: U) => ReactNode;
-  onRowClick?: (user: U) => void;
+interface Props {
+  users: (Watcher | Contributor)[];
```

Comment:
I feel like `users` might be a bit confusing here, since the real `User` object is nested inside either `Watcher` or `Contributor`, so the calls to access it would look a bit bizarre: `users[0].user`. I like how you used `member` on line 30, so maybe this could also be renamed to `members`?